### ISSUE TITLE
Restore from backup fails on RHEL 7.6 setup

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#17799]
Version: Master

Setup:
Cluster Nodes: RHEL 7.6 with docker 1.13

Steps:
1. Create a 3 node custom cluster (having all 3 roles) with RHEL 7.6 nodes with docker 1.13
2. Create few workloads 
3. Create a backup using (`Backup now` option from UI)
4. Restore the cluster from one of the backup.

Restore fails on a RHEL setup with the below error:
```
Failed to download etcd snapshot from backup server [a.b.c.d], exit code [1]: time="2019-01-31T23:00:07Z" level=fatal msg="open /etc/kubernetes/ssl/kube-ca.pem: no such file or directory"

```
Cluster is stuck in Updating state

**Comment by moelsayed** [comment]
A regression caused by a fix for #17521, reverted for now.

**Comment by soumyalj** [comment]
Tested with master build from Feb 4th. 
Steps:

1. Create a 3 node custom cluster (having all 3 roles) with RHEL 7.6 nodes with docker 1.13
2. Create few workloads
3. Create a backup using (Backup now option from UI)
4. Restore the cluster from one of the backup.
Restore succeeded with no data loss. New workloads were created successfully.

Also, the etcd cluster was broken by deleting etcd containers from two of the nodes. 
Performed restore from a backup. Restore succeeded with all data intact.
